### PR TITLE
Bug fix: Sentry

### PR DESCRIPTION
**Pull request by tuz666**
During the latest Tag related change we had to update Node.
Most probably this Node version bump caused a compatibility issue with the previously working and well configured Sentry lib.
The failure was recorded in #300 and it was resolved temporarily by the manual removal of Sentry.
This change contains a Sentry lib update and the removal of some unnecessary lines from its config.
The changes were tested on the dev server and after that I also verified it on the prod.

### CODE REVIEW

File: app/nuxt.config.js [hunk 1]
```diff
@@ -72,11 +72,6 @@ export default {
   },
   sentry: {
     dsn: process.env.SENTRY_DSN,
-    publish_release: true,
```
Comment (gammaw) [inline app/nuxt.config.js:75]: Do we need this removal for the fix to work? Any idea what these lines exactly do?

File: app/package.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
     "@babel/core": "^7.0.0-0",
     "@babel/plugin-proposal-optional-chaining": "^7.14.5",
     "@nuxtjs/axios": "^5.13.6",
-    "@nuxtjs/sentry": "^5.1.5",
-    "@sentry/tracing": "^7.62.0",
```
Comment (gammaw) [inline app/package.json:19]: Any idea what this is and why it needs to be removed?

File: app/package.json [hunk 3]
```diff
@@ -15,8 +15,8 @@
...
+    "@nuxtjs/sentry": "^7.5.0",
+    "@sentry/webpack-plugin": "^1.20.1",
```
Comment (gammaw) [inline app/package.json:19]: Why is this needed?

**Review by gammaw — COMMENTED** [review]
Thanks @tuz666! Allow me some questions before we move on.

File: app/package.json (hunk 2 above)
Comment (tuz666) [reply app/package.json:19]: @sentry/tracing is deprecated and no longer needed from @nuxtjs/sentry v7.1.12
https://github.com/nuxt-community/sentry-module/releases/tag/v7.1.12

File: app/nuxt.config.js (hunk 1 above)
Comment (tuz666) [reply app/nuxt.config.js:75]: This configuration would be used to monitor different releases, but it was wrongly configured (publishRelease instead of publish_release), during my investigation I tried to set this up properly but unfortunately I couldn't make it work.

File: app/package.json (hunk 3 above)
Comment (tuz666) [reply app/package.json:19]: The @nuxtjs/sentry version bump solved our issue.
@sentry/webpack-plugin package could be used for the publishRelease function if we want to implement it in the future.
https://sentry.nuxtjs.org/guide/releases

**Review by gammaw — APPROVED** [review]
Thanks @tuz666 for the detailed answers!
